test/UniversalRewardsDistributorTest.sol

```diff
@@ -71,6 +71,45 @@ contract UniversalRewardsDistributorTest is Test {
         assertEq(distributor.ipfsHash(), DEFAULT_IPFS_HASH);
     }
 
+    function testDistributionConstructorEmitsOwnerSet(address randomCreator) public {
+        bytes32 salt = bytes32(0);
+        bytes memory encodedParams = abi.encode(randomCreator, DEFAULT_TIMELOCK, DEFAULT_ROOT, DEFAULT_IPFS_HASH);
+        address urdAddress = _precomputeAddress(randomCreator, encodedParams, salt);
```

Comment:
However, I don't think it is useful here because you're testing the test (whereas you just care about events, you don't have to test that your test deploys with the correct salt and parameters)

Reply:
but you need the address before calling the constructor for expectEmit(urdAddress)

Reply:
Right, nice!